qemu/tests/virtio_console.py

```diff
@@ -1331,8 +1333,8 @@ def _tmigrate(use_serialport, no_ports, no_migrations, blocklen, offline):
             tmp = "%d data sent; " % threads[0].idx
             for thread in threads[1:]:
                 tmp += "%d, " % thread.idx
-            logging.debug("test_migrate: %s data received and verified",
-                          tmp[:-2])
+            test.log.debug("test_migrate: %s data received and verified",
+                           tmp[:-2])
             i += 1
             time.sleep(2)
 
```

Comment:
For L1342, fix the issue of missing `test.log.info` ?

Reply:
The same as the above.